Source/Core/DolphinQt/TAS/WiiTASInputWindow.cpp

```diff
@@ -268,6 +220,125 @@ WiiTASInputWindow::WiiTASInputWindow(QWidget* parent, int num) : TASInputWindow(
   UpdateExt(ext);
 }
 
+void WiiTASInputWindow::SetOrientationAndTriggerBoxes()
+{
+  m_remote_orientation_box = new QGroupBox(tr("Wii Remote Orientation"));
+
+  const QKeySequence remote_x_shortcut_key_sequence = QKeySequence(Qt::ALT + Qt::Key_Q);
+  const QKeySequence remote_y_shortcut_key_sequence = QKeySequence(Qt::ALT + Qt::Key_W);
+  const QKeySequence remote_z_shortcut_key_sequence = QKeySequence(Qt::ALT + Qt::Key_E);
+
+  auto* remote_x_label = new QLabel(QStringLiteral("%1 (%2").arg(
+      tr("X"), remote_x_shortcut_key_sequence.toString(QKeySequence::NativeText)));
+  auto* remote_y_label = new QLabel(QStringLiteral("%1 (%2").arg(
+      tr("Y"), remote_y_shortcut_key_sequence.toString(QKeySequence::NativeText)));
+  auto* remote_z_label = new QLabel(QStringLiteral("%1 (%2").arg(
+      tr("Z"), remote_z_shortcut_key_sequence.toString(QKeySequence::NativeText)));
+
+  QBoxLayout* remote_x_layout = new QHBoxLayout;
+  remote_x_layout->addWidget(remote_x_label);
+  QBoxLayout* remote_y_layout = new QHBoxLayout;
+  remote_y_layout->addWidget(remote_y_label);
+  QBoxLayout* remote_z_layout = new QHBoxLayout;
+  remote_z_layout->addWidget(remote_y_label);
+
+  SliderValuePair* remote_x_sliderValuePair =
```

Comment:
```suggestion
  SliderValuePair* remote_x_slider_value_pair =
```

Ditto for the ones below